javascript/javascript3/week3/ScreenShots/script.js

```diff
@@ -0,0 +1,254 @@
+// initializing variables
+// Decided not to hide crudcrudKEY anywhere - it is valid for only 24 hours
+const screenshotKEY = "af35d9bda9msh1d5d601753355fcp1d5ca1jsnc37e78a50099";
```

Comment:
Javascript files are accessible from the client side. Of course in these exercises there is no risk in keeping them visible, just have in mind that this cannot happen in projects that go to production.

Reply:
Martin, thanks for the remark. We have had a discussion about it during one of the classes. As I concluded, the proper way to "hide" the sensitive data is to put it to the .env file and then use across ones code. But to have it properly done one has to install Node.JS with specific npm, which I didn't at the moment of writing this code. In future I'm going to use .env